server/src/repositories/usersRepository.js

```diff
@@ -9,4 +9,59 @@ export const usersRepository = {
 
     return user;
   },
+
+  signup: async (email, github_oauth_id) => {
+    const connection = database.connection.getRepository(Users);
+    const savedUser = await connection.save({
+      email,
+      github_oauth_id,
+      github_oauth_token: "",
+    });
+  },
+
+  updateGithubOauthToken: async (github_oauth_id, github_oauth_token) => {
+    const connection = database.connection.getRepository(Users);
+    const user = await connection.findOne({
+      where: { github_oauth_id, github_oauth_id },
```

Comment:
`github_oauth_id`를 두번 받는 건가요??

Reply:
다시보니까 
```suggestion
      where: { github_oauth_id: github_oauth_id },
```
이렇게 하실려고 한걸까요?

Reply:
아 아래꺼가 맞습니다. 오타를 냈네요. 😨